src/common/options.cc

```diff
@@ -786,6 +786,12 @@ std::vector<Option> get_global_options() {
     .set_flag(Option::FLAG_NO_MON_UPDATE)
     .set_description(""),
 
+    Option("thp", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
might as well make this a dev option, no reason a regular user should ever change this

Reply:
done